### PR TITLE
Add the function to save video file

### PR DESCRIPTION
**Pull request by kdh0324**
We can save video file, but can't select file directory and name yet

### CODE REVIEW

**Comment by kdh0324** [comment]
We can't see progressbar....

**Comment by kdh0324** [comment]
Solve every error about saving video

**Comment by SangbumChoi** [comment]
> Solve every error about saving video

I will check about it nice work

After this task try to think about clicking event on the videoplayer or GPU setting on your computer

**Comment by SangbumChoi** [comment]
There is a error when playing saved video. I expect that this issue is coming from codec issue
@kdh0324 

Is there any solution for this?

Error code is 0xc10100be

**Comment by kdh0324** [comment]
In my computer, there isn't any error when playng saved video.
I don't know if this is the correct solution, but please try this. (https://support.microsoft.com/ko-kr/help/842180/cannot-play-back-the-file-the-format-is-not-supported-error-80040265-e)

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .idea/
 
 # ignore cached files
+/__pycache__/
```
[the remark this answers is not in the document]
Comment (SangbumChoi) [reply .gitignore]: The reason: if the development is done, cache file is related with the pycharm edition

File: Recorder/RecordApp.py [hunk 2]
```diff
@@ -47,57 +49,74 @@ def initUI(self):
...
-                        print(ex)
-            else:
-                errorbox = QMessageBox()
-                errorbox.warning(self, "Error Message", "Cannot read frame.", QMessageBox.Ok)
-                self.running = False
-                self.cap.release()
-                self.close()
-                break
+        self.timer.timeout.connect(self.stream)
+        self.timer.start(50)
```
[the remark this answers is not in the document]
Comment (SangbumChoi) [reply Recorder/RecordApp.py]: self.timer.start is related with frame per second so the reason of 50 is to set as 0.05s per one frame

File: VideoSave/VideoSave.py [hunk 3]
```diff
@@ -1,60 +1,53 @@
...
-        self.out.write(frame)
-        cv2.imshow('frame', frame)
+        cap.release()
+        out.release()
 
 
 if __name__ == "__main__":
     app = QApplication(sys.argv)
-    videoSave = VideoSave('input.mp4')
+    videoSave = VideoSave(r'C:\Users\kimda\Desktop\github\Team_Island\VideoSave\input.mp4')
```
[the remark this answers is not in the document]
Comment (SangbumChoi) [reply VideoSave/VideoSave.py]: because utf encoding we have to put r infront of directory or \\

this purpose was for saving test

**Pull request merged**